Packages/MIES/MIES_SweepFormula_Helpers.ipf

```diff
@@ -12,6 +12,74 @@
 
 static StrConstant SFH_WORKING_DF = "FormulaData"
 
+Function SFH_GetArgumentAsNumeric(variable jsonId, string jsonPath, string graph, string opShort, variable argNum, [variable defValue])
```

Comment:
I think some brief function docu would be helpful.
(also for `SFH_GetArgumentAsWave`)